adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -856,7 +856,7 @@ public void testRefreshTokenPositive() throws IOException, InterruptedException,
 
         final String response2 = "{\"id_token\":\""
                 + TEST_IDTOKEN
-                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"10\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
+                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"3600\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
```

Comment:
There are two reason I set it to 3600.
1. One hour is the default expiration time, which is the same with the mock in iOS.
2. I encountered the problem when I did the debugging. 10 seconds is way too short when I debugging the test, which caused unnecessary effort costs.

Thanks.